src/node_api.gyp

```diff
@@ -1,7 +1,8 @@
 {
   'targets': [
     {
-      'target_name': 'nothing'
+      'target_name': 'nothing',
+      'type': 'static_library'
```

Comment:
Is this part of the intended change?  Does not seem related to adding the script to check if modules are N-API or not.

Reply:
It kind of is. We need this so that node-addon-api might stop creating `nothing.node` files in other projects. I did change the script to ignore `nothing.node` files, but that's kind of a kludge. There's nothing stopping module authors from calling their module `nothing`, and the script would ignore it.

I'll PR it separately.